README.md

```diff
@@ -31,37 +36,255 @@ npm install @uploadcare/upload-client --save
 
 ## Usage
 
+### High Level API
+
+To get access to High Level API you need to create 
+an instance of `UploadClient` and provide settings to him:
```

Comment:
To access the High-Level API, you need to create an instance of `UploadClient` providing the necessary settings. Specifying `YOUR_PUBLIC_KEY` is mandatory: it points to the specific Uploadcare project,